lib/capitano.coffee

```diff
@@ -27,16 +27,18 @@ exports.globalOption = (options) ->
 	option = new Option(options)
 	exports.state.globalOptions.push(option)
 
-exports.execute = (args) ->
+exports.execute = (args, callback = _.noop) ->
 	command = exports.state.getMatchCommand(args.command)
 
 	if not command?
 		return exports.defaults.actions.commandNotFound(args.command)
 
 	try
-		command.execute(args)
+		command.execute(args, callback)
 	catch error
-		return exports.defaults.actions.onError(error)
+		return callback(error)
```

Comment:
@Page- Fixed. I'll await before merging for you to take a look if you don't mind for future PRs :)